CONTRIBUTING.md

```diff
@@ -150,20 +150,6 @@ tools:
   $ pyflakes path/to/module.py
```

Comment:
This should also say,
```
$ pip install flake8
$ flake8 path/to/module.py

Reply:
Alright. I'll make the required change.

> No flake8 warnings, check with:
> ```
> $ pip install flake8
> $ flake8 path/to/module.py
> ```

Does the above seem correct?

Reply:
Yes, sound good (and replace the pyflakes section with this).  Thanks.

**jnothman**  was referring to https://help.github.com/articles/closing-issues-using-keywords/